strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/ArtifactResolutionServiceImpl.java

```diff
@@ -48,24 +49,22 @@
     private LayoutProviderRegistry layoutProviderRegistry;
 
     @Override
-    public RepositoryInputStream getInputStream(String storageId,
-                                                String repositoryId,
-                                                String artifactPath)
+    public RepositoryInputStream getInputStream(RepositoryPath path)
         throws IOException,
```

Comment:
I believe the three exceptions below are actually not thrown. (Or, at least, I believe this was the case in the `GroupRepositoryProvider`).